lemur/common/celery.py

```diff
@@ -366,3 +371,83 @@ def sync_source_destination():
     current_app.logger.debug("Completed Syncing AWS destinations and sources")
     red.set(f'{function}.last_success', int(time.time()))
     metrics.send(f"{function}.success", 'counter', 1)
+
+
+@celery.task()
+def certificate_reissue():
+    """
+    This celery task reissues certificates which are pending reissue
+    :return:
+    """
+    function = f"{__name__}.{sys._getframe().f_code.co_name}"
+    current_app.logger.debug(f"{function}: reissuing certificates")
+    cli_certificate.reissue(None, True)
+    current_app.logger.debug(f"{function}: reissuance completed")
```

Comment:
Can we log in dictionary format? This will prepare for the eventual JSONification of the logs. Example:

{"function": function,
"message": "reissuing certificates"
}